app/controllers/assignments_controller.rb

```diff
@@ -39,7 +39,9 @@ def create
   end
 
   def update
+    confirmation = @assignment.confirmation
     if @assignment.update(assignment_params)
+      delete_reminder if @assignment.confirmation && @assignment.confirmation != confirmation
```

Comment:
You can also do this in the model in an `after_update` callback, Rails adds helpers for all attributes so you can just call `confirmation_changed?` on the model (still works in after callbacks too)

See http://craftingruby.com/posts/2014/01/13/callbacks-and-dirty-objects-in-rails.html

Reply:
Thanks! I will do it a bit differently because I got this warning:
DEPRECATION WARNING: The behavior of `attribute_changed?` inside of after callbacks will be changing in the next version of Rails. The new return value will reflect the behavior of calling the method after `save` returned (e.g. the opposite of what it returns now). To maintain the current behavior, use `saved_change_to_attribute?` instead.